funcx_sdk/funcx/sdk/login_manager/manager.py

```diff
@@ -78,14 +78,29 @@ def run_login_flow(
 
         do_link_auth_flow(self._token_storage, scopes)
 
-    def logout(self) -> None:
+    def logout(self, resource_server: str = None) -> bool:
+        """
```

Comment:
(In line with the other comment, this would clearly need to return more than `bool`.)

Reply:
Ugh; I see a referenced a different comment that ... GitHub seems to have lost?!  Did I not actually click "Add single comment" button?  Ugh; will rewrite it.

Reply:
Ah, it happened again; the diff changed under me, but this time I was smart enough to copy it before clicking the button.  "Thaaaaanks, GitHub."